app/DashBoard/Presenters/templates/Blocks/favicons.latte

```diff
@@ -0,0 +1,8 @@
+{block favicon}
+<link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
```

Comment:
@landsman nebyla by lepší to hodit do komponenty? a adresy neobsahují {$basePath}